pandas/tseries/tests/test_partial_string_indexes.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+import pandas.util.testing as tm
```

Comment:
put it with the existing tests
pls following the existing testing methodologies 

iow don't create your own helpers it's not standard